doc/nrf/known_issues.rst

```diff
@@ -326,6 +326,22 @@ Bluetooth LE
 
 .. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
 
+NCSDK-8224: Callbacks for "security changed" and "pairing failed" are not always called
+  The pairing failed and security changed callbacks are not called when the connection is disconnected during the pairing procedure or the required security is not met.
+
+  **Workaround:** Application should use the disconnected callback in addition to handle pairing failed.
+
+.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+NCSDK-8223: GATT requests might deadlock RX thread
+  GATT requests might deadlock the RX thread when all TX buffers are taken by GATT requests and the RX thread tries to allocate a TX buffer for a response.
+  This causes a deadlock because only the RX thread releases the TX buffers for the GATT requests.
+  The deadlock is resolved by a 30 second time-out, but the ATT bearer cannot transmit without reconnecting.
```

Comment:
```suggestion
  The deadlock is resolved by a 30 second timeout, but the ATT bearer cannot transmit without reconnecting.
```

Reply:
@ru-fu is it really "time-out"?

Reply:
No, it's time-out according to our style guide.

Reply:
https://en.wiktionary.org/wiki/timeout
vs
https://en.wiktionary.org/wiki/time-out

"timeout" is the usual spelling in Computer Science AFAICT

Reply:
https://projecttools.nordicsemi.no/confluence/display/TECHDOC/List+of+terms%2C+abbreviations%2C+and+spelling+rules

Reply:
https://github.com/nrfconnect/sdk-nrf/pull/3905#discussion_r575053141 :) going with this one